pkg/services/ngalert/notifier/channels/utils.go

```diff
@@ -32,3 +35,13 @@ func getTitleFromTemplateData(data *template.Data) string {
 	}
 	return title
 }
+
+// RemoveSystemLabels removes the labels added by Grafana from the alerts
```

Comment:
I think it's best if we centralise this removal - In the end, I don't think those "system labels" are something we'd want and I want to avoid us having to scramble through multiple files to remove them.

How do you feel if we move it to the alert reception function? Can be part of a separate PR if you'd prefer (but I still think we need to fix the CI anyways)

Reply:
I moved it to `PutAlerts` now